.gitmodules

```diff
@@ -12,3 +12,6 @@
 [submodule "third_party/abseil"]
 	path = third_party/abseil
 	url = https://github.com/abseil/abseil-cpp.git
+[submodule "third_party/cctz"]
```

Comment:
Ugh.  Why does abseil not install cctz?  And how do we make sure we update both at the same time (and to a consistent revision).

Reply:
They do, they bring it into their [`WORKSPACE`](https://github.com/abseil/abseil-cpp/blob/cf6ab6bb2b800fae859ccc735f398d22a7336a00/WORKSPACE#L9-L14) file, presumably for a Bazel build:

> ```
> # CCTZ (Time-zone framework).
> http_archive(
>     name = "com_googlesource_code_cctz",
>     urls = ["https://github.com/google/cctz/archive/master.zip"],
>     strip_prefix = "cctz-master",
> )
> ```

Since we're not using Bazel, we need to get the dependency ourselves.

Note that they don't pin it to a specific version, they just get it from the latest `master` at the time of build, so we don't have to worry about getting a specific "compatible" version since they don't do it themselves.

Reply:
Ack.  Thanks!  Still wondering about why we need absl::Mutex though.

Reply:
That's a question for @DanielMahu. :-)

Reply:
I think this was just inertia on my part, std::mutex should be great too. In which case we can forget about linking with Abseil (and cctz).